### PR TITLE
Interop

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Review by davidwengier — APPROVED** [review]

**Review by jinujoseph — APPROVED** [review]

File: src/EFTools/DesignXmlCore/DesignXmlCore.csproj [hunk 1]
```diff
@@ -63,16 +63,9 @@
     <Reference Include="Microsoft.VisualStudio.Shell.$(VisualStudioVersion15)" />
     <Reference Condition="'$(VisualStudioVersion)' == '16.0'" Include="Microsoft.VisualStudio.Shell.Framework" />
     <Reference Include="Microsoft.VisualStudio.Package.LanguageService.$(VisualStudioVersion15)" />
-    <Reference Include="Microsoft.VisualStudio.OLE.Interop" />
+    <Reference Include="Microsoft.VisualStudio.Interop" />
```
Comment (sharwell) [inline src/EFTools/DesignXmlCore/DesignXmlCore.csproj:66]: ❔ This should be `PackageReference` right?

Comment (ryanbrandenburg) [reply src/EFTools/DesignXmlCore/DesignXmlCore.csproj:65]: This repo is using the old-school project system with `packages.config`, so `PackageReference` isn't recognized.

**Comment by ryanbrandenburg** [comment]
Build failed on the Publish step due to permissions issues.

File: src/EFTools/DesignXmlCore/DesignXmlCore.csproj (hunk 1 above)
Comment (ryanbrandenburg) [reply src/EFTools/DesignXmlCore/DesignXmlCore.csproj]: After much pain and many iterations I ended up moving this to `PackageReference` anyway

**Pull request merged**